src/Numberdrome.js

```diff
@@ -1,7 +1,33 @@
 'use strict';
 
 class Numberdrome {
-  // write code here
+  constructor() {
+    this.numbers = [];
+  }
+  addNumber(n) {
+    this.numbers.push(n);
+  }
+  sum() {
+    return this.numbers.reduce((acc, x) => acc + x);
+  }
+  product() {
+    return this.numbers.reduce((acc, x) => acc * x);
+  }
```

Comment:
Плохие названия переменных.
```suggestion
  sum() {
    return this.numbers.reduce((acumulator , number) => acumulator + number);
  }
  product() {
    return this.numbers.reduce((acumulator , number) => acumulator * number);
  }
```